data.json

```diff
@@ -0,0 +1,39 @@
+[
```

Comment:
What's the plan/workflow for keeping this file up-to-date?  Every time a new file is available, we have to update `hash`?  In some cases it looks like we'll have to update `file` too, like `AddressNC_2014.gdb`.  That's unfortunate.